Recipely/Recipely/Services/NetworkService/DTO/Hit.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Hit.swift
+//  Recipely
+//
+//  Created by Tixon Markin on 13.03.2024.
+//
+
+import Foundation
+
+/// Структура, представляющая отдельный элемент массива `hits` в JSON-ответе.
+struct Hit: Codable {
+    /// Объект типа `DishDTO`, содержащий информацию о конкретном блюде.
+    let recipe: DishDTO
```

Comment:
```suggestion
    let recipe: DishDTO
```
не соответсвуют названиям